prow/scripts/cluster-integration/kyma-aks-long-lasting.sh

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail  # Fail a pipe if any sub-command fails.
+
+VARIABLES=(
+    RS_GROUP
+    REGION
+    AZURE_SUBSCRIPTION_ID
+    AZURE_SUBSCRIPTION_APP_ID
+    AZURE_SUBSCRIPTION_SECRET
+    AZURE_SUBSCRIPTION_TENANT
+    KYMA_PROJECT_DIR
+    INPUT_CLUSTER_NAME
+    GOOGLE_APPLICATION_CREDENTIALS
+    CLOUDSDK_DNS_ZONE_NAME
+    CLOUDSDK_CORE_PROJECT
+    KYMA_ALERTS_CHANNEL
+    KYMA_ALERTS_SLACK_API_URL
+    SLACK_CLIENT_WEBHOOK_URL
+    STABILITY_SLACK_CLIENT_CHANNEL_ID
+    SLACK_CLIENT_TOKEN
+    TEST_RESULT_WINDOW_TIME
+    DOCKER_PUSH_REPOSITORY
+    DOCKER_PUSH_DIRECTORY
+)
+
+discoverUnsetVar=false
+
+for var in "${VARIABLES[@]}"; do
+    if [ -z "${!var}" ] ; then
+        echo "ERROR: $var is not set"
+        discoverUnsetVar=true
+    fi
+done
+if [ "${discoverUnsetVar}" = true ] ; then
+    exit 1
+fi
+
+# INIT ENVIRONMENT VARIABLES
+export TEST_INFRA_SOURCES_DIR="${KYMA_PROJECT_DIR}/test-infra"
+export TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS="${TEST_INFRA_SOURCES_DIR}/prow/scripts/cluster-integration/helpers"
+export KYMA_SOURCES_DIR="${KYMA_PROJECT_DIR}/kyma"
+export KYMA_SCRIPTS_DIR="${KYMA_SOURCES_DIR}/installation/scripts"
+export KYMA_RESOURCES_DIR="${KYMA_SOURCES_DIR}/installation/resources"
+
+export DOCKER_PUSH_PROJECT="kyma-installer"
```

Comment:
I cannot see usage of those variable, document if needed or remove

Reply:
👍